src/main/kotlin/org/jetbrains/bio/genome/AnnotationsConfig.kt

```diff
@@ -147,11 +147,15 @@ object AnnotationsConfig {
                 emptyMap()
             }
 
-            val names = when (val names = genomeAttrs["names"]) {
-                is String -> listOf(names)
-                is List<*> -> names.map { it.toString() }
-                else -> listOf(build) // fallback for errors
-            }
+            /* names = build + ucsc_alias + aliases, with duplicates removed */
+            val names = arrayListOf(build)
+            val ucscAlias = genomeAttrs["ucsc_alias"] as? String
+            ucscAlias?.let { if (build != it) names.add(it) }
+            names.addAll(when (val aliases = genomeAttrs["aliases"]) {
```

Comment:
`aliases` could also contain build name or `ucsc_alias` if user didn't get the conventions